saleor/graphql/csv/types.py

```diff
@@ -0,0 +1,46 @@
+import graphene
+from graphene import relay
+from graphene_federation import key
+from graphql_jwt.exceptions import PermissionDenied
+
+from ...core.permissions import AccountPermissions
+from ...csv import models
+from ..core.connection import CountableDjangoObjectType
+from .enums import JobStatusEnum
+
+
+@key(fields="id")
+class Job(CountableDjangoObjectType):
+    url = graphene.String(description="The URL of field to download.")
+    status = JobStatusEnum(description="Job status.")
+
+    class Meta:
+        description = "Represents job data"
```

Comment:
A little bit more explicit:

```suggestion
        description = "Represents a data export job."
```